Vagrantfile

```diff
@@ -5,4 +5,5 @@ Vagrant.configure("2") do |config|
   config.vm.provider "virtualbox" do |v|
     v.memory = 1024
   end
+  config.vm.network "forwarded_port", guest: 5000, host: 5000, host_ip: "127.0.0.1"
```

Comment:
@SaarCohen93 I see this line as a duplicate, we do not need it.

Reply:
deleted.

Reply:
I do not see it deleted.